config/dev.yml

```diff
@@ -19,6 +19,8 @@ hint:
   oauth2_client_id: VAULT:secret/hint/oauth2/auth0:id
   oauth2_client_secret: VAULT:secret/hint/oauth2/auth0:secret
   oauth2_client_url: VAULT:secret/hint/oauth2/auth0:url
+  oauth2_client_adr_url: VAULT:secret/hint/oauth2/auth0:adr-url
+  oauth2_client_audience: VAULT:secret/hint/oauth2/auth0:audience
```

Comment:
Are these secrets? Do they need to be in vault or should we just have 
```
oauth2_client_adr_url: https://naomi-preview.dide.ic.ac.uk/resource-server
oauth2_client_audience: naomi
```

Reply:
> Are these secrets? Do they need to be in vault or should we just have
> 
> ```
> oauth2_client_adr_url: https://naomi-preview.dide.ic.ac.uk/resource-server
> oauth2_client_audience: naomi
> ```

Maybe not that secret but dynamic. I can remove them from the vault.